Makefile-docker

```diff
@@ -67,12 +67,11 @@ help_submake:
 
 initialize_db:
 	rm -rf ./user-media/* ./tmp/*
-	python manage.py reset_db
+	python manage.py reset_db --noinput
 	python manage.py syncdb --noinput
 	python manage.py loaddata initial.json
 	python manage.py import_prod_versions
 	schematic --fake src/olympia/migrations/
-	python manage.py createsuperuser
```

Comment:
if you're removing `createsuperuser` from `initialize_db` it needs to be inserted somewhere else -  `initialize` calls `initialize_db` and that's what developers (us!) call to get everything set up at first.

Reply:
Good point. Should I just put all the initialize stuff for the ui tests in its own make command? I just didn't want to have 2 commands doing the same thing essentially.

Reply:
I think I'd prefer that. It's separate enough that creating a new command kinda makes sense to me